framework/src/Volo.Abp.AspNetCore.Mvc.UI.Theme.Shared/Pages/Shared/Components/AbpPageSearchBox/Default.cshtml

```diff
@@ -0,0 +1,11 @@
+﻿
+<form method="post" class="searchForm">
+    <div class="input-group">
+        <input class="form-control filterText" placeholder="Search"/>
```

Comment:
* Rename `filterText` to `page-search-filter-text`
* Make placeholder localized